core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -2974,7 +2974,7 @@ protected function isValid($strFile)
 			throw new AccessDeniedException('Invalid file name "' . $strFile . '" (hacking attempt).');
 		}
 
-		if (Validator::isInsecurePath($strFolder))
+		if (null === $strFolder || Validator::isInsecurePath($strFolder))
```

Comment:
```suggestion
		if (null !== $strFolder && Validator::isInsecurePath($strFolder))
```

I think this would be the correct fix to get rid of the PHP warning.

Reply:
@aschempp Are you still working on this PR?